linkerd/app/admin/src/server.rs

```diff
@@ -201,7 +201,7 @@ where
 mod tests {
     use super::*;
     use http::method::Method;
-    use std::time::Duration;
+    use tokio::time::Duration;
```

Comment:
i believe `tokio`'s `Duration` is [just a re-export of `std::time::Duration`](https://docs.rs/tokio/latest/tokio/time/index.html#reexports), so we really don't *need* to change this...it's fine that we changed it, but if we wanted to keep the diff small, we could remove it...

i believe the re-export exists mainly so that `Instant` and `Duration` can be imported from the same place?